src/main/asciidoc/reference/auditing.adoc

```diff
@@ -18,4 +18,10 @@ import org.springframework.data.neo4j.annotation.EnableNeo4jAuditing;
 public class Neo4jConfiguration {}
 ----
 
-Please refer to the <<auditing,auditing>> section of the shared Spring Data reference for further details.
\ No newline at end of file
+[NOTE]
+If you are working with custom ids by setting them directly or using your own `IdStrategy`, the auditing support is limited.
+In this case it is not possible for Spring Data Neo4j to figure out if an entity is new or already exists in the database,
```

Comment:
```suggestion
It is not possible for Spring Data Neo4j to figure out if an entity is new or already exists in the database if the id field is set through a custom strategy.
```

Reply:
Custom strategy, as mentioned in the sentence before, is just one of the two possibilities to provide an id besides the internal (native id). If we would only focus on custom strategy (the term) here it might sound more confusing. But yes, also my expression could lead to the same assumption the other way round.